### DISCUSSION TITLE
I designed some icons for freeplane

### DISCUSSION THREAD

**Discussion by 1466852206** [freeplane/freeplane#1788] [Ideas]
<img width="1920" alt="Frame 8" src="https://github.com/freeplane/freeplane/assets/56283390/08dbc3f9-28c4-4e1c-91e7-cc4f52604235">
I hope it can be used. It's all open source. I hope the interface can be improved. Please use figma to open the source file.
[icon.zip](https://github.com/freeplane/freeplane/files/15012387/icon.zip)

**Comment by Olifair** [comment]
Does anybody remember why there is a bee hanging around in the start-up screen ? What has the bee to do with Freeplane ? Actually does anybody remember why the "plane" was introduced when Freeplane split from Freemind ? Should it have been "Freebee" ?

**Comment by euu2021** [comment]
> I designed some icons for freeplane

The icons and splash screen you designed look good. 

I know that changing the icon set is painful, but I hope that at least this splash screen could be adopted. I prefer it way more than the current one.

> Does anybody remember why there is a bee hanging around in the start-up screen ? What has the bee to do with Freeplane ? Actually does anybody remember why the "plane" was introduced when Freeplane split from Freemind ? Should it have been "Freebee" ?

I know that, in the past, the bee was in the main icon, too. See https://sourceforge.net/p/freeplane/discussion/758437/thread/32151df94b/
